infrastructure/game-client-startup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Script to install applications for the Game Client VM 
+
+export DEBIAN_FRONTEND=noninteractive
+
+curl -1sLf 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-l4t/setup.deb.sh' | sudo -E bash
+sudo apt install moonlight-qt
+sudo apt install xfce4 xfce4-goodies -y
```

Comment:
All these apt installs could be one line, would be a bit faster.

Also, can we install `xpra` as well please? Would be nice to have the latest version too: https://github.com/Xpra-org/xpra/wiki/Download#-for-debian-based-distributions